openedx/features/ucsd_support/urls.py

```diff
@@ -0,0 +1,16 @@
+"""
+Map urls to the relevant view handlers
+"""
+
+from django.conf.urls import url
+from .views import EmailSupportVIew
+
+app_name = "ucsd_support"
```

Comment:
The question of whether this app should be support email only is still undecided. However, if they request more features later on, we will change it then. It is fine for now.

Reply:
ok